### PR TITLE
CORE-1762: disable data uploads when the storage limit has been exceeded.

### PR DESCRIPTION
**Pull request by slr71**

### CODE REVIEW

**Comment by slr71** [comment]
I think this is ready for review.

**Review by psarando — APPROVED** [review]
LGTM 👍 
I just had some minor style suggestions, but :shipit: when ready.

File: src/components/data/toolbar/UploadMenuBtn.js [hunk 1]
```diff
@@ -59,6 +69,16 @@ function UploadMenuBtn(props) {
...
 
+    const uploadButtonIcon = uploadsEnabled ? (
+        <>
+            <UploadIcon />
+        </>
+    ) : (
+        <>
+            <BlockIcon />
+        </>
+    );
```
[the remark this answers is not in the document]
Comment (slr71) [reply src/components/data/toolbar/UploadMenuBtn.js]: 👍

File: src/components/data/toolbar/UploadMenuBtn.js [hunk 2]
```diff
@@ -75,7 +95,7 @@ function UploadMenuBtn(props) {
                 onClick={onUploadMenuClick}
                 aria-haspopup={true}
                 aria-controls={uploadMenuId}
-                startIcon={<UploadIcon />}
+                startIcon={uploadButtonIcon}
```
[the remark this answers is not in the document]
Comment (slr71) [reply src/components/data/toolbar/UploadMenuBtn.js]: Ah! I searched for a cleaner way to do that, but gave up after a while. Thanks! 👍

**Comment by slr71** [comment]
Thanks for the review!

**Pull request merged**